completions/_gal

```diff
@@ -6,7 +6,7 @@ _gal() {
 
   case $state in
     (guardfile)
-      guardfiles=("${(f)$(ls $HOME/code/dotfiles/guardfiles | sd '^run_' '' | sd '.rb$' '')}")
+      guardfiles=("${(f)$(ls $HOME/code/dotfiles/guardfiles | sd '^run_' '' | sd '\.rb$' '')}")
```

Comment:
(Unrelated change; I noticed that this period should be escaped in the regex pattern.)